src/api/settings.js

```diff
@@ -0,0 +1,22 @@
+import { getInstance } from 'd2'
+import { onError } from './index'
+import { SYSTEM_SETTINGS } from '../modules/settings'
```

Comment:
If Jan is correct in his comment below that `extractUserSettings` isn't used, then I suggest just getting rid of `modules/settings` and use `keyGatherAnalyticalObjectStatisticsInDashboardViews` directly in this file. If more settings are needed in the future, then we can expand this.

Reply:
Sounds good. Removed modules/settings.